pandas/tests/frame/indexing/test_mask.py

```diff
@@ -90,6 +90,19 @@ def test_mask_dtype_bool_conversion(self):
         result = bools.mask(mask)
         tm.assert_frame_equal(result, expected)
 
+    def test_mask_pos_args_deprecation(self):
+        # https://github.com/pandas-dev/pandas/issues/41485
+        df = DataFrame(np.random.randn(5, 5))
+        cond = df > 0
+        other = DataFrame(np.random.randn(5, 3))
+
+        msg = (
+            r"Starting with Pandas version 2\.0 all arguments of mask except for the "
+            r"arguments 'self' and 'cond' will be keyword-only"
+        )
```

Comment:
the message will need updating, there's been some changes in `deprecate_nonkeyword_arguments`

Reply:
Have made the changes and removed the message entirely.